src/backend/lima.ts

```diff
@@ -300,7 +300,7 @@ export default class LimaBackend extends events.EventEmitter implements VMBacken
     case State.STOPPED:
     case State.ERROR:
     case State.DISABLED:
-      await this.kubeBackend.stop();
+      await this.kubeBackend.destroy();
```

Comment:
Is this not too drastic? Do we need a second parameter to `setState` to indicate we actually want to do this?

Reply:
Perhaps we should rename `destroy`? I think this (before the refactor) was just clearing the k8s client anyway, so maybe just call it `destroyClient`?